satip/app.py

```diff
@@ -65,7 +65,7 @@
 )
 @click.option(
     "--history",
-    default="60 minutes",
+    default="120 minutes",
```

Comment:
could you put this back to 60 mins as deafult

Reply:
I have put this back to 60 mins